receiver/jaegerreceiver/testdata/config.yaml

```diff
@@ -1,16 +1,37 @@
 receivers:
+  # The following demonstrates initializing the default jaeger receiver.
+  # By default all three protocols (grpc, thrift-http, thrift-tchannel)
+  # are enabled and the default endpoints are specified in factory.go
   jaeger:
+  # The following demonstrates specifying different ports.
+  # The Jaeger receiver only connects to ports and ignores addresses.
+  # However, an address is needed to specify for a proper configuration.
+  # Ex: `endpoint: "9876"` is incorrect
+  # Ex: `endpoint: "1.2.3.4:9876"` is correct
   jaeger/customname:
     protocols:
       grpc:
         endpoint: "127.0.0.1:9876"
-        disabled: true
       thrift-http:
         endpoint: "127.0.0.1:3456"
-        disabled: true
       thrift-tchannel:
         endpoint: "0.0.0.0:123"
 
+  # The following demonstrates disabling the receiver.
+  # All of the protocols need to be disabled for the receiver to be disabled.
+  # If only one protocol is disabled, that is ignored and the protocol is
```

Comment:
```suggestion
  # If only one protocol is disabled, that is ignored and the other protocols are
```

Reply:
@ccaraman now I am confuse about the behavior in this case :) could you please clarify? `other protocols` or `all protocols`?

Reply:
all protocols
I'm thinking of changing it to 
`If a subset of the protocols are disabled, the disabled flags are ignored and all protocols are enabled.` Is that clearer?

Reply:
So it's either all enabled or none? Is there a way to only enable one protocol? If not, why not just have one top-level `disabled` flag?

Reply:
It is a bug in the receiver code. It is noted in the issue #158 

with a detailed explanation of how that is happening in this comment
https://github.com/open-telemetry/opentelemetry-service/issues/158#issuecomment-521767782

A few lines below this comment i mention that it will be fixed by that issue.

Reply:
I see, thanks for the pointer.